skrub/_gap_encoder.py

```diff
@@ -857,7 +856,7 @@ def _handle_missing(self, X):
                 f"'zero_impute', got {self.handle_missing!r}. "
             )
 
-        missing_mask = _object_dtype_isnan(X)
+        missing_mask = pd.isna(X)
```

Comment:
I assume that we can make the same changes in `SimilarityEncoder` and `TargetEncoder`.
I would also expect to see a mention of the values considered as missing values in the docstring of the encoder since it is not straightforward.

Reply:
good point, thanks!